src/main/java/competition/subsystems/lights/LightsCommunicationSubsystem.java

```diff
@@ -108,7 +113,11 @@ public void periodic() {
 
         // Figure out what we want to send to the arduino
         if (!dsEnabled) {
-            currentState = LightsStateMessage.RobotDisabled;
+            if (autonomousCommandSelector.getCurrentAutonomousCommand() != null) {
```

Comment:
Do we have a DoNothingCommand as default?

Reply:
Good question, I only see `null` in the code at least as the default value of the selector command